arch/xtensa/src/esp32/esp32_wifi_adapter.c

```diff
@@ -4201,8 +4199,6 @@ int32_t esp_timer_start_periodic(esp_timer_handle_t timer, uint64_t period)
 {
   struct rt_timer_s *rt_timer = (struct rt_timer_s *)timer;
 
-  DEBUGASSERT(period <= UINT32_MAX);
```

Comment:
Shouldn't be replaced with UINT64_MAX ?

Reply:
@acassis isn't it always true for a uint64_t value?

Reply:
period must be <= UINT64_MAX.

Reply:
Yes, you are right, I missed that period is a uint64_t.